tests/functional/test_vcs_mercurial.py

```diff
@@ -0,0 +1,29 @@
+"""
+Contains functional tests of the Mercurial class.
+"""
+
+import os
+
+from pip._vendor.six.moves import configparser
+
+from pip._internal.utils.temp_dir import TempDirectory
+from pip._internal.vcs import RevOptions
+from pip._internal.vcs.mercurial import Mercurial
+
+
+def test_mercurial_switch_updates_config_file_when_found():
```

Comment:
I believe there is a pytest marker for hg tests?

Reply:
Yes. `tests.lib.needs_mercurial`

Reply:
Fixed, thanks